waspc/src/Wasp/AppSpec/Valid.hs

```diff
@@ -13,12 +13,15 @@ import Data.Maybe (isJust)
 import Wasp.AppSpec (AppSpec)
 import qualified Wasp.AppSpec as AS
 import Wasp.AppSpec.App (App)
+import qualified Wasp.AppSpec.App as AS.App
 import qualified Wasp.AppSpec.App as App
 import qualified Wasp.AppSpec.App.Auth as Auth
+import qualified Wasp.AppSpec.App.Db as AS.Db
 import Wasp.AppSpec.Core.Decl (takeDecls)
 import qualified Wasp.AppSpec.Entity as Entity
 import qualified Wasp.AppSpec.Entity.Field as Entity.Field
 import qualified Wasp.AppSpec.Page as Page
+import Wasp.Generator.ServerGenerator.JobGenerator (isPgBossUsed)
```

Comment:
I don't think we want AppSpec to depend on Generator -> we are trying to keep AppSpec above both Analyzer and Generator, not dependent on any of them.

I believe you have two options:
1. Move the logic for `isPgBossUsed` to AppSpec (here or somewhere else in AppSpec module tree). I believe you can do it, you don't need Generator.
2. Don't do the check here and throw an error in Generator instead.

Reply:
Ah good point. I can try to move `isPgBossUsed` into AppSpec instead. Thanks!